src/components/Shared/Filters/NullState/NullState.tsx

```diff
@@ -89,17 +99,8 @@ const NullState: React.FC<Props> = ({
       />
       {filtered ? (
         <>
-          <Typography variant="h5">
-            <Trans
-              defaults="You have {{count}} total {{page}}s"
-              values={{ count: totalCount, page }}
-            />
-          </Typography>
-          <Typography>
-            {t(
-              'Unfortunately none of them match your current search or filters.',
-            )}
-          </Typography>
+          <Typography variant="h5">{title}</Typography>
+          <Typography>{paragraph}</Typography>
           <Box display="flex" mt={1}>
             <Button variant="contained" onClick={() => changeFilters({})}>
               {t('Reset All Search Filters')}
```

Comment:
Angular doesn't have any buttons here. I don't think you need them. This isn't really the place to clear filters or add a new contact so I think it will just confuse the user with too many things.
<img width="1314" alt="Screenshot 2024-08-21 at 12 48 57 PM" src="https://github.com/user-attachments/assets/16eb455d-802c-4cdd-a01b-6e6a61bdd98a">

Reply:
Fixed in future PR.